.github/workflows/pull-requests.yml

```
@@ -233,7 +234,7 @@ jobs:
         run: yarn up --exact graphql@${{ matrix.graphql-version }}
       - name: Run @neo4j/graphql integration tests and move coverage
         run: |
-          yarn test:int --coverage
+          yarn test:int --shard=${{ matrix.shard }}/4 --coverage
```

With the suggestion from above - what do you think?

```suggestion
          yarn test:int --shard=${{ matrix.shard }} --coverage
```